src/mmw/js/src/analyze/templates/catchmentWaterQualityTable.html

```diff
@@ -12,17 +12,43 @@
             <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
                 Total TSS (kg/ha)
             </th>
+            <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
+                Avg TN (mg/l)
+            </th>
+            <th class="text-right" data-sortable="true" data-sorter="window.noDataSort">
+                Avg TN (mg/l)
```

Comment:
Yeah! I thought I had pushed it but it looks like I did not. commit ff054cb should have it.